src/project/cached-project-module-spec.ts

```diff
@@ -1,17 +1,17 @@
+import { caching } from 'cache-manager';
 import { expect } from 'chai';
 import 'reflect-metadata';
 
-import * as cacheManager from 'cache-manager';
 import { Cache } from '../shared/cache';
 import CachedProjectModule from './cached-project-module';
 
 describe('cached-project-module', () => {
 
   function setupCache(): Cache {
-    return cacheManager.caching({
+    return caching({
       store: 'memory',
       ttl: 1000,
-    }) as Cache;
+    }) as {} as Cache;
```

Comment:
This is by design, but admittedly suboptimal. We have just rewritten the Cache interface with promises.